src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_COUNT = 100;
+    private Random random;
+    private ColorSupplier colorSupplier;
+
+    public Ball getRandomBall() {
+        Color color = Color.valueOf(colorSupplier.getRandomColor());
```

Comment:
Lets bring our `Color` representation to a single type, both `ColorSupplier.getRandomColor()` return type and `Ball.color` field type should be the same